tests/common.go

```diff
@@ -128,7 +133,7 @@ func ValidateVolumes(ctx *scheduler.Context) {
 	context("For validation of an app's volumes", func() {
 		var err error
 		Step(fmt.Sprintf("inspect %s app's volumes", ctx.App.Key), func() {
-			err = Inst().S.InspectVolumes(ctx)
+			err = Inst().S.InspectVolumes(ctx, 10*time.Second, 2*time.Second)
```

Comment:
Make these constants

Reply:
Also `ValidatePersistentVolumeClaim()` in sched-ops previously had a `validatePVCTimeout` of 5 minutes. This is now reduced to 10 seconds. 

As part of this change, ensure that the timeouts and intervals remaining the same. At some point, we will reduce the timeouts to more stricter limits.

Reply:
@harsh-px  I'm keeping max of timeout for all internal operation running, for eg in inspectVolume `ValidatePVCsForStatefulSet` has max timeout of 15 minutes so I will pass 15min timeout to inspectVolumes from tests that way we will not keep any hardcore values at `drivers/sched/k8s` side

Reply:
15 minutes is too much to allow a PVC to be bound. Just 5 minutes. In scheduler driver `InspectVolumes` implementation, use `Statefulset Replicas x timeout`, as the timeout for `ValidatePVCsForStatefulSet` since a statefulset has replica number of PVCs. For e.g If the timeout is given as 5 minutes and we have a statefulset with replica 3, all PVCs and Snapshot will be tested with timeout of 5 minutes. Statefulset PVCs will be tested with timeout of 5 * 3 minutes.

Reply:
sounds good, I have also made similar changes for ValidatePod in WaitForRunning which requires 10 Minute timeout